notifier.go

```diff
@@ -0,0 +1,15 @@
+package backoff
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/micrologger"
+)
+
+func NewNotifier(l micrologger.Logger, ctx context.Context) func(error, time.Duration) {
+	return func(err error, d time.Duration) {
+		l.LogCtx(ctx, "level", "warning", "message", fmt.Sprintf("retrying backoff in '%s' due to error", d.String()), "stack", fmt.Sprintf("%#v", err))
```

Comment:
Are you aware that the `caller` will always be `notifier.go:13`? Is this information good enough?

Reply:
The stack shows you where it is coming from so I think that works.